### ISSUE TITLE
Accessibility | Theme Switcher

### ISSUE THREAD

**Issue by f4-lex** [Frachtwerk/essencium-frontend#833]
**Describe the bug**
The Theme switcher is not operable using keyboard navigation. You can open it, but you can't choose a theme.

**Expected behaviour**
The theme switch should be operable using keyboard navigation.

- [ ] update `MIGRATION.md`
- [ ] update docs
